meta-lmp-base/recipes-support/docker-image-preload/docker-image-preload/docker-image-preload.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Docker container images preloading
+
```

Comment:
Please remove this empty line.

Reply:
Why? I put it there deliberately, IMHO  it makes 'code' more readable.

Reply:
Just so we can separate the blocks via an empty line.